lib/generators/auxiliary_rails/templates/rubocop/rubocop_auxiliary_rails_template.yml

```diff
@@ -42,3 +42,9 @@ Style/IfUnlessModifier:
 
 Style/NegatedIf:
   EnforcedStyle: postfix
+
+Style/EmptyCaseCondition:
+  Enabled: false
+
+Style/RescueStandardError:
+  Enabled: false  
```

Comment:
Why disable instead of switching to `implicit`?

Reply:
How do you propose to do it?

Reply:
Just like they do in any RuboCop config - https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml#L3884